build.sbt

```diff
@@ -160,6 +162,7 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     )
   )
   .nativeSettings(
+    scalacOptions += "-P:scalanative:genStaticForwardersForNonTopLevelObjects",
```

Comment:
Shouldn't it be different for Scala 3?
```scala
(CrossVersion.partialVersion(scalaVersion.value) match {
    case Some((2, _)) => "-P:scalanative:genStaticForwardersForNonTopLevelObjects"
    case _ => "-scalanative-genStaticForwardersForNonTopLevelObjects"
  })
```

Reply:
Nope, that doesn't work. Has to be like it is currently in the PR. The tests being green are the proof :)

Reply:
'-P' stands for prefix for compiler settings. In case of Scala.js the compiler plugin setting in Scala 2 was elevated to standalone setting in Scala 3 after Scala.js backend become a part of compiler. 
Scala Native is not going to become part of compiler at least until 1.0